src/main/java/org/fundacionjala/sfdc/util/CommonActions.java

```diff
@@ -209,6 +242,4 @@ public static void closeMessageLighting() {
             System.out.println("Message not displayed.");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)